src/main/java/com/labs/introtoprogramming/lab4/image/formats/bmp/BMPParser.java

```diff
@@ -0,0 +1,56 @@
+package com.labs.introtoprogramming.lab4.image.formats.bmp;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+
+public class BMPParser {
+
+  public static int parseHeader(byte[] headerInfo) throws Exception {
```

Comment:
Should it really by public?
Static is arguable is as well.

Applies to all other methods in this class.